v1/src/main/java/com/google/cloud/teleport/spanner/ddl/InformationSchemaScanner.java

```diff
@@ -1231,4 +1241,58 @@ private void listSequenceOptionsPostgreSQL(
           .endSequence();
     }
   }
+
```

Comment:
Can you please add a comment on what is a placeholder descriptor?

Reply:
Done